README.md

```diff
@@ -1,13 +1,24 @@
-# Project Name
+# Chat Bot
+This project, which was realized in a pair programming setup, was about the fundamental syntax and concepts fo JavaScript. The task was to build a Chat Bot, which in our case, tells the user where to go on vacation based on the users answers to questions.
 
-Replace this readme with your own information about your project.
+## The problem
+The first step was about understanding the given code and how it functions. After being able to respond with the users name we started to inspect other chatbots and see how they are set up to understand the concept of JS better. Our first approach was to collect answers to three different questions and put all those answers in one if statement, which we couldn't figure out how to make it work. So we took a step back and made the if statements after each question. 
+During the coding process we followed the given iteration and checked after each interaction if it was working like we expected it to. When we stumbled over errors we used the inspect-tool to understand what was going wrong. Also we got great help from classmates on the #help channel and also stackoverflow. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+We set up a time schedule for the week to always work together on the code. The main plan was setting up the questions and answers and then we dealed with each section as we went along, how to proceed to next questions, if statement etcetera. 
 
-## The problem
+If we had more time we would have worked on the cold-, warm-button and make them disappear after clicking. We couldn't figure that out. So now it stays until the next buttons appear. 
+We also tried to make the code more neat but did not know what part to put together so if more time we would have made that work with fewer lines of code (for example the setTimeOut, which should go easier)
+Our problem not being able to move forward when stuck has been depended on finding a solution by google or comparing other students code but not being able to implement it in our own code. We really want to work on this. 
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+When stuck we were struggling to implement a solution even though we got code examples from students or examples we found on google. We want to work on this.
 
-## View it live
+Feedback  we very much appreciate:
+- How to make the buttons disappear after clicking one. (we tried  form.innerHTML = ""; and inputWrapper.innerHTML ="" with no sucess)
```

Comment:
You can create for example class:
```
 .hidden {
  display: none;
}
``` 
and add it to your buttons inside click handler. https://www.w3schools.com/howto/howto_js_add_class.asp